src/main/java/org/opentripplanner/updater/stoptime/GtfsRealtimeHttpTripUpdateSource.java

```diff
@@ -55,6 +60,9 @@ public void configure(Graph graph, JsonNode config) throws Exception {
         }
         this.url = url;
         this.feedId = config.path("feedId").asText();
+        this.connectionTimeout = config.path("connectionTimeoutMills").asInt();
```

Comment:
This is repeating code that appears elsewhere in the pull request. Rather than adding this functionality to one specific polling graph updater, it should ideally be added to all of them via the shared superclass org.opentripplanner.updater.PollingGraphUpdater. Please refactor to place the timeout configuration in the most general superclass that handles HTTP polling.